microsphere-multiactive-spring-cloud-loadbalancer/src/main/java/io/microsphere/multiple/active/zone/spring/cloud/loadbalancer/CustomizedLoadBalancerAutoConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package io.microsphere.multiple.active.zone.spring.cloud.loadbalancer;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.cloud.loadbalancer.annotation.LoadBalancerClientSpecification;
+import org.springframework.cloud.loadbalancer.support.LoadBalancerClientFactory;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Collections;
+
+/**
+ * {@link ZonePreferenceServiceInstanceListSupplier} Auto-Configuration Class
+ *
+ * @author <a href="mailto:warlklown@gmail.com">Walklown<a/>
+ * @since 1.0.0
+ */
+@Configuration(proxyBeanMethods = false)
+@ConditionalOnClass(LoadBalancerClientFactory.class)
```

Comment:
* Probably a typo. It's still @ConditionalOnClass(LoadBalancerClientFactory.class).
* @LoadBalancerAutoConfiguration is not necessary because it will not wait until the Configuration stage to load, but will be loaded in advance when loading lbRestClientPostProcessor. But I still accept the suggestion that maybe there will be a different loading environment.

![image](https://github.com/microsphere-projects/microsphere-multiactive/assets/22697462/97c35aa7-4820-40c3-b679-4f9398b28f26)
![image](https://github.com/microsphere-projects/microsphere-multiactive/assets/22697462/693148c9-aaa1-4286-8f4d-ff5ddc8d44ae)